dotcom-rendering/src/web/components/Card/Card.tsx

```diff
@@ -307,80 +306,79 @@ export const Card = ({
+								format={format}
+							/>
+						</AvatarContainer>
+					) : (
 						<img src={imageUrl} alt="" role="presentation" />
-					</ImageWrapper>
-				)}
+					)}
+				</ImageWrapper>
 				<ContentWrapper
+					imageType={imageType}
 					imageSize={imageSize}
 					imagePosition={imagePosition}
 				>
-					<Flex>
-						<HeadlineWrapper>
-							<CardHeadline
-								headlineText={headlineText}
-								format={format}
+					<HeadlineWrapper>
+						<CardHeadline
+							headlineText={headlineText}
+							format={format}
+							containerPalette={containerPalette}
+							size={headlineSize}
+							sizeOnMobile={headlineSizeOnMobile}
+							showQuotes={showQuotes}
+							kickerText={
+								format.design === ArticleDesign.LiveBlog
+									? 'Live'
+									: kickerText
+							}
+							showPulsingDot={
+								format.design === ArticleDesign.LiveBlog ||
+								showPulsingDot
+							}
+							showSlash={
+								format.design === ArticleDesign.LiveBlog ||
+								showSlash
+							}
+							byline={byline}
+							showByline={showByline}
+							isDynamo={isDynamo}
+						/>
+						{starRating !== undefined ? (
+							<StarRatingComponent rating={starRating} />
+						) : null}
+						{(format.design === ArticleDesign.Gallery ||
+							format.design === ArticleDesign.Audio ||
+							format.design === ArticleDesign.Video) &&
+						mediaType ? (
+							<MediaMeta
 								containerPalette={containerPalette}
-								size={headlineSize}
-								sizeOnMobile={headlineSizeOnMobile}
-								showQuotes={showQuotes}
-								kickerText={
-									format.design === ArticleDesign.LiveBlog
-										? 'Live'
-										: kickerText
-								}
-								showPulsingDot={
-									format.design === ArticleDesign.LiveBlog ||
-									showPulsingDot
-								}
-								showSlash={
-									format.design === ArticleDesign.LiveBlog ||
-									showSlash
-								}
-								byline={byline}
-								showByline={showByline}
-								isDynamo={isDynamo}
+								format={format}
+								mediaType={mediaType}
+								mediaDuration={mediaDuration}
 							/>
-							{starRating !== undefined ? (
-								<StarRatingComponent rating={starRating} />
-							) : null}
-							{(format.design === ArticleDesign.Gallery ||
-								format.design === ArticleDesign.Audio ||
-								format.design === ArticleDesign.Video) &&
-							mediaType ? (
-								<MediaMeta
-									containerPalette={containerPalette}
-									format={format}
-									mediaType={mediaType}
-									mediaDuration={mediaDuration}
-								/>
-							) : undefined}
-						</HeadlineWrapper>
-						{imageType === 'avatar' && !!avatarUrl && (
-							<Hide when="above" breakpoint="tablet">
-								<AvatarContainer>
-									<Avatar
-										imageSrc={avatarUrl}
-										imageAlt={byline ?? ''}
-										containerPalette={containerPalette}
-										format={format}
-									/>
-								</AvatarContainer>
-							</Hide>
```

Comment:
The previous logic solved the problem for where to position the avatar on mobile by using two avatars, show/hiding them at different breakpoints. We no longer do this, instead we share the `flex-direction` logic used by the main image